### DISCUSSION TITLE
Flood for Transmission does not officially support Transmission 2.94

### DISCUSSION THREAD

**Discussion by johman10** [haugene/docker-transmission-openvpn#1945] [General]
Hi!

[An issue was opened on Flood for Transmission](https://github.com/johman10/flood-for-transmission/issues/243) informing me that labels don't work with this docker image. I just wanted to document/inform everyone here as well that Flood for Tranmission doesn't officially support Transmission 2.94 which is the Transmission version of the latest Docker image. This means that Flood for Transmission might work perfectly fine for your use case, but some things might not work great.

If anyone bumps into issues there you might want to follow option 3 documented here: https://github.com/haugene/docker-transmission-openvpn/discussions/1937 to still switch to Transmission 3.0.

Please let me know if anything is unclear or if there is any feedback on this. :)

**Comment by johman10** [comment]
For the devs, is there any plan of switching back to Transmission >3.0? Are you waiting for a new release to iron out some of the issues your users ran in to?

**Reply by pkishino** [reply]
for now we haven't discussed moving to 3.x due to a number of issues such as memory leaks etc..at some future point I am sure we will
